KubeArmor/types/types.go

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+//Types package is an extention of the crd types.
```

Comment:
```suggestion
// Package ypes is an extension of the CRD types.
```
Pleasing the linter :)